<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signupWidget</class>
 <widget class="QWidget" name="signupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>237</width>
     <height>189</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="usrLabel">
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QRadioButton" name="questionerBtn">
      <property name="text">
       <string>出题者</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="pwdLabel">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>昵称</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QRadioButton" name="playerBtn">
      <property name="text">
       <string>闯关者</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="usrLineEdit"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="returnBtn">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="okBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>立即注册</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="pwdLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="roleLabel">
      <property name="text">
       <string>角色</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="pwdLabel_2">
      <property name="text">
       <string>确认密码</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="pwdLineEdit_2">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usrLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>pwdLineEdit</tabstop>
  <tabstop>pwdLineEdit_2</tabstop>
  <tabstop>playerBtn</tabstop>
  <tabstop>questionerBtn</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>returnBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
